### PR TITLE
patterns for storage volume optimisation

### PR DESCRIPTION
**Pull request by greenhsu123**

### CODE REVIEW

File: patterns/rightsizing-storage-volumes.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Rightsizing storage volume
+
+## Version
+TBD
```
Comment (dubrie) [inline patterns/rightsizing-storage-volumes.md:4]: Update to version 1.0

**Review by dubrie — COMMENTED** [review]
Initial review completed

File: patterns/rightsizing-storage-volumes.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Problem
+From both an embodied carbon angle and an energy proportionality angle, it's better to delete idle storage resources or maximise storage utilisation, so the storage layer is optimised for the task. 
+
+From the embodied carbon angle, the unused capacity of underutilised storage could be more efficiently used by another task or process. From the energy proportionality angle, two storage running at low utilisation rates will consume more energy than one storage running at a high utilisation rate.
+
+## Solution
+Reduce idle storage resources by 
+1. Delete any unused storage resource
+2. Set a retention policy on storage resources to automate the deletion of unused storage resources 
+3. Move storage resources away from underutilised storage
```
Comment (jawache) [inline patterns/rightsizing-storage-volumes.md:27]: Would be good to add two patterns here, one for managed storage resources such as S3 one for unmanaged storage resources such as drives and long term storage solutions.

File: patterns/rightsizing-storage-volumes.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Cloud, Storage, Cloud Engineer, Small
+
+## Problem
+From both an embodied carbon angle and an energy proportionality angle, it's better to delete idle storage resources or maximise storage utilisation, so the storage layer is optimised for the task. 
+
+From the embodied carbon angle, the unused capacity of underutilised storage could be more efficiently used by another task or process. From the energy proportionality angle, two storage running at low utilisation rates will consume more energy than one storage running at a high utilisation rate.
+
+## Solution
+Reduce idle storage resources by 
+1. Delete any unused storage resource
```
Comment (jawache) [inline patterns/rightsizing-storage-volumes.md:25]: Suggest splitting this to another pattern.

File: patterns/rightsizing-storage-volumes.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+2. Set a retention policy on storage resources to automate the deletion of unused storage resources
```
Comment (jawache) [inline patterns/rightsizing-storage-volumes.md:26]: Suggest splitting this to another pattern.

File: patterns/rightsizing-storage-volumes.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+From the embodied carbon angle, the unused capacity of underutilised storage could be more efficiently used by another task or process. From the energy proportionality angle, two storage running at low utilisation rates will consume more energy than one storage running at a high utilisation rate.
+
+## Solution
+Reduce idle storage resources by 
+1. Delete any unused storage resource
+2. Set a retention policy on storage resources to automate the deletion of unused storage resources 
+3. Move storage resources away from underutilised storage
+
+## SCI Impact
```
Comment (jawache) [inline patterns/rightsizing-storage-volumes.md:29]: Each spit off pattern might have different effects on the SCI impact

**Review by jawache — CHANGES_REQUESTED** [review]
Looks great, need a few changes before we can move forward.

File: patterns/rightsizing-storage-volumes.md (hunk 2 above)
Comment (greenhsu123) [reply patterns/rightsizing-storage-volumes.md]: We could, but they both would have v similar solution. How about we use tags to encapsulate that the pattern can be for both managed and unmanaged storage?

**Review by jawache — APPROVED** [review]
LGTM

**Pull request merged**